packages/workflow/src/Expression.ts

```diff
@@ -54,6 +62,22 @@ export class Expression {
 		return `[${typeName}: ${JSON.stringify(value)}]`;
 	}
 
+	extendSyntax(bracketedExpression: string): string {
+		if (!hasExpressionExtension(bracketedExpression)) return bracketedExpression;
+
+		const unbracketedExpression = bracketedExpression.replace(/(^\{\{)|(\}\}$)/g, '').trim();
```

Comment:
In here we do a global replacement so for expressions like `{{ $json.name }} <{{ $json.email }}>` where the goal is to output a string such as `John Doe <johndoe@example.com>` possibly wouldn't work, right? I think we should think about splitting the original strings into blocks that correspond to expressions and use this those blocks as the foundation. WDYT?

Reply:
`extendSyntax()` is already operating on expression unit, i.e. `/(^\{\{)|(\}\}$)/g` matches on each expression block separately.
![image](https://user-images.githubusercontent.com/44588767/179697333-164bf494-f957-4440-bb30-e2977808913d.png)